rest/taskana-rest-spring-test/src/test/java/pro/taskana/rest/ClassificationDefinitionControllerIntTest.java

```diff
@@ -337,6 +337,38 @@ public void testImportParentAndChildClassificationWithKey() throws IOException {
         assertNotEquals(wrongParentCl.getClassificationId(), childCl.getParentId());
     }
 
+    @Test
+    public void testChangeParentByImportingExistingClassification() throws IOException, InterruptedException {
+        ClassificationSummaryResource child1 = this.getClassificationWithKeyAndDomain("L110105", "DOMAIN_A");
+        assertEquals("L11010", child1.getParentKey());
+        ClassificationSummaryResource child2 = this.getClassificationWithKeyAndDomain("L110107", "DOMAIN_A");
+        assertEquals("L11010", child2.getParentKey());
+
+        child1.setParentId("CLI:100000000000000000000000000000000002");
+        child1.setParentKey("L10303");
+        String withNewParent = objMapper.writeValueAsString(child1);
+
+        child2.setParentId("");
+        child2.setParentKey("");
+        String withoutParent = objMapper.writeValueAsString(child2);
+
+        List<String> clList = new ArrayList<>();
+        clList.add(withNewParent);
+        clList.add(withoutParent);
+
+        ResponseEntity<String> response = importRequest(clList);
+        assertEquals(HttpStatus.OK, response.getStatusCode());
+        Thread.sleep(10000);
```

Comment:
Why 10 seconds? I think this is unnecessary. Tested it locally with 10ms. Runs fine.

Reply:
You're right!
Thanks for your testing!